### ISSUE TITLE
Error with string index keys

### ISSUE THREAD

**Issue by Nvveen** [rtk-incubator/rtk-query-codegen#57]
I am using an OpenAPI spec with parameters that are valid string indices, but not valid object property names, such as `created_at>` and `created_at<`.

An example spec:
```json
{
  "openapi": "3.0.0",
  "info": {
    "title": "title",
    "description": "description",
    "contact": {
      "name": "Name",
      "url": "http://example.com",
      "email": "email@example.com"
    },
    "version": "1.0"
  },
  "servers": [
    {
      "url": "/v1"
    }
  ],
  "paths": {
    "/foo/bar": {
      "get": {
        "tags": [
          "Foo - Bar"
        ],
        "summary": "Summary",
        "operationId": "foo_bar",
        "parameters": [
          {
            "name": "created_at>",
            "in": "query",
            "description": "Filter on date",
            "required": false,
            "schema": {
              "type": "string"
            }
          },
          {
            "name": "created_at<",
            "in": "query",
            "description": "Filter on date",
            "required": false,
            "schema": {
              "type": "string"
            }
          }
        ],
        "responses": {
          "200": {
            "description": "Success response",
            "content": {
              "application/json": {
                "schema": {
                  "type": "array",
                  "items": {
                    "$ref": "#/components/schemas/FooBar"
                  }
                }
              }
            }
          }
        }
      }
    }
  },
  "components": {
    "schemas": {
      "FooBar": {
        "properties": {
          "amount": {
            "type": "string"
          }
        },
        "type": "object"
      }
    }
  }
}
```

After which I get the following error:

```
SyntaxError: Type parameter declaration expected. (8:73)
   6 |     endpoints: build => ({
   7 |         fooBar: build.query<FooBarApiResponse, FooBarApiArg>({
>  8 |             query: queryArg => ({ url: `/foo/bar`, params: { created_at<: queryArg.createdAt } })
     |                                                                         ^
   9 |         })
  10 |     })
  11 | });
    at e (/tmp/test/rtk-query-codegen/node_modules/prettier/parser-typescript.js:1:322)
    at Object.parse (/tmp/test/rtk-query-codegen/node_modules/prettier/parser-typescript.js:1:3072720)
    at Object.parse (/tmp/test/rtk-query-codegen/node_modules/prettier/index.js:13625:19)
    at coreFormat (/tmp/test/rtk-query-codegen/node_modules/prettier/index.js:14899:14)
    at format (/tmp/test/rtk-query-codegen/node_modules/prettier/index.js:15131:14)
    at /tmp/test/rtk-query-codegen/node_modules/prettier/index.js:57542:12
    at Object.format (/tmp/test/rtk-query-codegen/node_modules/prettier/index.js:57562:12)
    at Object.prettify (/tmp/test/rtk-query-codegen/lib/utils/prettier.js:53:21)
    at /tmp/test/rtk-query-codegen/lib/bin/cli.js:90:39 {
  loc: { start: { line: 8, column: 73 } },
  codeFrame: '   6 |     endpoints: build => ({\n' +
    '   7 |         fooBar: build.query<FooBarApiResponse, FooBarApiArg>({\n' +
    '>  8 |             query: queryArg => ({ url: `/foo/bar`, params: { created_at<: queryArg.createdAt } })\n' +
    '     |                                                                         ^\n' +
    '   9 |         })\n' +
    '  10 |     })\n' +
    '  11 | });'
}
```

**Comment by phryneas** [comment]
That should get fixed by #52 (I'll get to that soon now that RTKQ is out, I promise!), but might end up with very weird property & hook names. Can you try that one out?

**Comment by Nvveen** [comment]
For that minimal reproduction, it seems to work fine :)

```ts
import { createApi } from "@reduxjs/toolkit/query/react";
import { fetchBaseQuery } from "@reduxjs/toolkit/query";
export const api = createApi({
  baseQuery: fetchBaseQuery({ baseUrl: "/v1" }),
  tagTypes: [],
  endpoints: (build) => ({
    fooBar: build.query<FooBarApiResponse, FooBarApiArg>({
      query: (queryArg) => ({
        url: `/foo/bar`,
        params: {
          "created_at>": queryArg["created_at>"],
          "created_at<": queryArg["created_at<"],
        },
      }),
    }),
  }),
});
export type FooBarApiResponse = /** status 200 Success response */ FooBar[];
export type FooBarApiArg = {
  /** Filter on date */
  "created_at>"?: string;
  /** Filter on date */
  "created_at<"?: string;
};
export type FooBar = {
  amount?: string;
};
export const { useFooBarQuery } = api;
```

**Comment by phryneas** [comment]
Resolved with 0.4.0
